### DISCUSSION TITLE
open source

### DISCUSSION THREAD

**Discussion by Xionse** [Software-Savants/.github#4] [Q&A] [answered]
what are some of the negative effects of open source?

**Comment by AnthonyByansi** [comment, marked as answer]
While open source software can have many benefits, there are also some potential negative effects to consider:

Lack of support: Some open source software may not have the same level of support as proprietary software, which can make it more difficult to get help if you encounter problems or have questions.

Compatibility issues: Open source software may not always be compatible with proprietary software or hardware, which can limit its use in some situations.

Lack of funding: Because open source software is usually developed by volunteers, it may not have the same level of funding as proprietary software. This can limit the resources available for development and maintenance.

Complexity: Some open source software may be more complex to use than proprietary software, which can be a barrier for some users

**Reply by Addax101** [reply]
ooh yeah sure

**Reply by Xionse** [reply]
ooh yeah opefully get the sense now
